tests/unit/test-util/test.artifacts.js

```diff
@@ -55,4 +55,71 @@ describe('prepareArtifactsDir', () => {
     }
   ));
 
+  it('throws an UsageError when it lacks permissions to stat the directory',
+    () => withTempDir(
+    function(tmpDir) {
+      if (process.platform === 'win32') {
```

Comment:
@saintsebastian can you add an inline comment here (and in the following two other skipped tests) which explains why they are skipped and link the related issue?
something like: 
```// Skipped on windows because the mkdir permissions do not have any effect (web-ext#ISSUENUMBER)```